build.gradle

```diff
@@ -111,20 +111,32 @@ subprojects {
     apply plugin: 'java'
     apply plugin: 'maven-publish'
 
+    def nettyVersion = libs.versions.netty.get()
+
     configurations.all {
         resolutionStrategy {
             // Force specific versions of transitive dependencies to handle CVEs
-            force 'net.minidev:json-smart:2.5.2'
-            force 'xalan:xalan:2.7.3'
+            force "com.fasterxml.jackson.core:jackson-databind:${libs.versions.jackson.get()}"
+            force "com.google.guava:guava:${libs.versions.guava.get()}"
+            force "commons-io:commons-io:2.14.0"
+            force "io.netty:netty-codec:${libs.versions.netty.get()}"
```

Comment:
it looks like `nettyVersion` is already a defined variable -- is there a reason why we repeat the call from line 114 for each of these?

Reply:
Oh thanks, removing